code-reviews/example1/index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>All About Broccoli</title>
+    <link rel='stylesheet' type='text/css' href='styles.css'>
+</head>
+<body class='body'>
+    <div class='main'>
```

Comment:
Why did u enclose the content in a div tag? I don't think it affects the code in any way